portfolio/src/main/webapp/index.html

```diff
@@ -20,9 +20,9 @@ <h1>Sofia Riley</h1>
       <p> On campus I am involved with the Assocation of Women in Computing, the Penn State Equestrian Team, the Girls Who Code College Loop, and the Women in Engineering Program.</p>
       <p> I am a Technology Lead for the Girls Who Code College Loop and a Facilitator for the study groups organized by the Women in Engineering Program.</p>
       
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <p>Click here to get a phrase:</p>
+      <button onclick="showPhrase()">Phrase</button>
+      <div id="phrase-container"></div>
       <img style ='position:absolute; top:25px; left: 25px; width:300px; height:250px' src= "images/profile.jpg">
```

Comment:
I like how everything is looking. I don't have a lot of knowledge in HTML or CSS, but I think you can put the img style on the stylesheet. Other than that, everything is great.

Reply:
+1 to what Claudia said